src/stems/stem-tracker.js

```diff
@@ -8,66 +8,85 @@ var Stem = require("./stem");
  * A stem is a widget insertion button. It signifies a potential widget.
  * The tracking utility manages the absolute positioning of these stems,
  * as well as their addition and removal from the DOM.
+ *
+ * Exclusions can be added so that certain nodes don't have stems created for
+ * them. The strings in the exclusions array should be lowercase.
```

Comment:
Document what exclusions are.  From the comment I'm guessing that it's an array of strings.  Tag names? Element ids?  (I know I can read the code, but we don't want to have to make people do that)